ember/app/ui/cases/detail/index/controller.js

```diff
@@ -26,79 +26,124 @@ export default class CasesDetailIndexController extends Controller {
-            return [...workItems, workItem];
-          }
+    const newestWorkItem = configuredWorkItems.reduce(
+      (prevWorkItem, currentWorkItem) =>
+        new Date(prevWorkItem.createdAt) > new Date(currentWorkItem.createdAt)
+          ? prevWorkItem
+          : currentWorkItem
+    );
 
+    const alwaysDisplayedWorkItem = configuredWorkItems.reduce(
+      (workItems, workItem) => {
+        if (
+          !Object.keys(ENV.APP.caluma.alwaysDisplayedAnswers).includes(
+            workItem.task.slug
+          )
+        ) {
           return workItems;
-        }, [])
-        /*
-         * This filters the answers of the workItem document,
-         * only the configured answers in displayedAnswers should remain
-         * and based on another configured questions answer the answer will be filtered or not
-         */
-        .map((workItem) => {
-          return workItem.document.answers.edges.reduce(
-            (filteredAnswers, answer, _, answers) => {
-              Object.keys(ENV.APP.caluma.displayedAnswers).forEach(
-                (taskSlug) => {
-                  if (!workItem.task.slug.includes(taskSlug)) {
-                    return;
-                  }
-
-                  const decision = answers.find(
-                    (a) => a.node.question.slug === `${taskSlug}-decision`
-                  );
-                  const value =
-                    decision.node[`${decision.node.__typename}Value`];
-
-                  if (
-                    answer.node.question.slug ===
-                    ENV.APP.caluma.displayedAnswers[taskSlug][value]
-                  ) {
-                    filteredAnswers.push({
-                      label: answer.node.question.label,
-                      value: answer.node[`${answer.node.__typename}Value`],
-                    });
-                  }
-                }
-              );
-
-              return filteredAnswers;
-            },
-            []
-          );
-        })
-        .flat()
+        }
+
+        const duplicateIndex = workItems.findIndex(
+          (item) => item.task.slug === workItem.task.slug
+        );
+
+        if (duplicateIndex === -1) {
+          return [...workItems, workItem];
+        } else if (
+          new Date(workItems[duplicateIndex].createdAt) <
+          new Date(workItem.createdAt)
+        ) {
+          workItems.splice(duplicateIndex, 1);
+          return [...workItems, workItem];
+        }
+
+        return workItems;
+      },
+      []
     );
+
+    return { newest: newestWorkItem, always: alwaysDisplayedWorkItem };
+  }
+
+  /*
+   * This filters the answers of the workItem document,
+   * only the configured answers in displayedAnswers should remain
+   * and based on another configured questions answer the answer will be filtered or not.
+   * Answers in alwaysDisplayedAnswers should always be displayed.
+   */
+  get remarks() {
+    const answersNewest =
```

Comment:
I would perhaps call this `newestAnswer`.

Reply:
Changed